common/lib/capa/capa/inputtypes.py

```diff
@@ -1800,10 +1802,10 @@ def extract_choices(element, i18n):
 
         for choice in element:
             if choice.tag != 'choice':
-                msg = u"[capa.inputtypes.extract_choices] {0}".format(
+                msg = Text("[capa.inputtypes.extract_choices] {0}").format(
                     # Translators: a "tag" is an XML element, such as "<b>" in HTML
-                    _("Expected a {expected_tag} tag; got {given_tag} instead").format(
-                        expected_tag=u"<choice>",
+                    Text(_("Expected a {expected_tag} tag; got {given_tag} instead")).format(
+                        expected_tag=Text("<choice>"),
```

Comment:
I personally don't think it is a bad thing to have the `Text()` here to make it clear that this HTML-like text is definitely supposed to be treated as plain text, but you are good either way.